package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "graphiql-monorepo",
-  "version": "independent",
+  "version": "0.0.0",
```

Comment:
this was incorrect 
![image](https://user-images.githubusercontent.com/7361780/205279326-e5010606-10bb-43ce-8ed6-a60be1d22895.png)

Reply:
@B2o5T changing this used to impact other tooling, but we haven't used `lerna publish` for a while, so it should be fine now